File: 1.Python/Program2.py
#   2. Write a program which contains one function named as ChkNum()
#   which accept one parameter as number.
#   If number is even then it should display “Even number” otherwise
#   display “Odd number” on console.
#   Input : 11
#   Output : Odd Number
#   Input : 8
#   Output : Even Number
def ChkNum(No):
       if No%2==0:
           print("Even number")
       else:
           print("Odd number")
def main():
    No=0
    print("Enter the one number:")
    No=int(input())
    ChkNum(No)
if __name__=="__main__":
    main()

File: 0.C_CPP_JAVA Programs/JAVA/Program2.java
/*2. Write Java program which accept N numbers from user and display all such elements which are divisible by 5.
Input :  N :   6
  Elements : 85 66 3 80 93 88
Output : 85 80*/
import java.lang.*;
import java.util.*;
class Displayn
{
    void Div(int size,int arr[])
    {
        for(int i=0;i<arr.length;i++)
        {
            if(arr[i]%5==0)
            {
                System.out.println(" " + arr[i]);
            }
        }
    }
}
class demo
{
    public static void main(String arg[])
    {
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the size:");
        int size=sobj.nextInt();

        int arr[] = new int[size];

        System.out.println("Enter the element:");
        for(int i=0;i<arr.length;i++)
        {
            arr[i] = sobj.nextInt();
        }

        Displayn dobj=new Displayn();

         dobj.Div(size,arr);
    }

}